crates/meroctl/src/cli/context/invite.rs

```diff
@@ -16,7 +16,9 @@ use crate::output::Report;
 #[derive(Debug, Parser)]
 #[command(about = "Create invitation to a context")]
 pub struct InviteCommand {
-    #[arg(
+    #[clap(
+        long = "context",
+        short = 'c',
```

Comment:
```suggestion
        long,
        short,
```

PS: feel free to break it up into it's own `#[arg(..)]` annotation